language/intro_prompt_design.ipynb

```diff
@@ -1,739 +1,1124 @@
+        "* Vertex AI Generative AI Studio\n",
+        "\n",
+        "Learn about [Vertex AI pricing](https://cloud.google.com/vertex-ai/pricing),\n",
+        "and use the [Pricing Calculator](https://cloud.google.com/products/calculator/)\n",
+        "to generate a cost estimate based on your projected usage."
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "3e663cb43fa0"
+      },
+      "source": [
+        "### Install Vertex AI SDK"
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "82ad0c445061"
+      },
+      "outputs": [],
+      "source": [
+        "!pip install google-cloud-aiplatform --upgrade --user"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "cebd6983cbad"
+      },
+      "source": [
+        "**Colab only:** Uncomment the following cell to restart the kernel or use the button to restart the kernel. For Vertex AI Workbench you can restart the terminal using the button on top."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "bea801acf6b5"
+      },
+      "outputs": [],
+      "source": [
+        "# # Automatically restart kernel after installs so that your environment can access the new packages\n",
+        "# import IPython\n",
+        "\n",
+        "# app = IPython.Application.instance()\n",
+        "# app.kernel.do_shutdown(True)"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "7a386d25fa8f"
+      },
+      "source": [
+        "### Authenticating your notebook environment\n",
+        "* If you are using **Colab** to run this notebook, uncomment the cell below and continue.\n",
+        "* If you are using **Vertex AI Workbench**, check out the setup instructions [here](https://github.com/GoogleCloudPlatform/generative-ai/tree/main/setup-env)."
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 1,
+      "metadata": {
+        "id": "1bd1dca8e9a7"
+      },
+      "outputs": [],
+      "source": [
+        "from google.colab import auth\n",
+        "auth.authenticate_user()"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "960505627ddf"
+      },
+      "source": [
+        "### Import libraries"
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "ue7q-YO3Scpp"
+      },
+      "source": [
+        "**Colab only:** Uncomment the following cell to initialize the Vertex AI SDK. For Vertex AI Workbench, you don't need to run this.  "
+      ]
+    },
+    {
+      "cell_type": "code",
+      "execution_count": 7,
+      "metadata": {
+        "id": "NGvWtLAyScpp"
+      },
+      "outputs": [],
+      "source": [
+        "import vertexai\n",
+        "\n",
+        "PROJECT_ID = \"ram-test-228600\"  # @param {type:\"string\"}\n",
```

Comment:
by default this should be empty. Since this is already a param, user will need to fill out for it to proceed

Reply:
Ok removed it